addon/webextension/_locales/en_US/messages.json

```diff
@@ -0,0 +1,33 @@
+{
+  "pageShotDescription": { "message": "Page Shot takes clips and screenshots from pages, and can save a permanent copy of a page." },
+  "pageShotAuthors": { "message": "Ian Bicking, Donovan Preston, and Bram Pitoyo" },
+  "toolbarButtonLabel": { "message": "Take a shot" },
+  "selectorInstructions": { "message": "Drag or click on the page to select a region. Press ESC to cancel." },
+  "selectorMyShots": { "message": "My Shots" },
+  "selectorSave": { "message": "Save" },
+  "selectorSaveVisible": { "message": "Save visible" },
+  "selectorSaveFullPage": { "message": "Save full page" },
+  "selectorCancel": { "message": "Cancel" },
+  "selectorDownload": { "message": "Download" },
+  "contextMenuText": { "message": "Create Page Shot" },
+  "notificationLinkCopied": { "message": "Link Copied" },
+  "notificationLinkCopiedDetails": {
+    "message": "The link to your shot has been copied to the clipboard. Press $PASTE_SYMBOL$-V to paste.",
+    "placeholders": {
+      "PASTE_SYMBOL": {
+        "content": "$1"
+       }
+    }
+  },
+  "serverError": { "message": "Page Shot is out of order." },
+  "requestErrorDetails": { "message": "Your shot was not saved. We apologize for the inconvenience. Try again soon." },
+  "connectionError": { "message": "Cannot connect to the Page Shot server." },
+  "connectionErrorDetails": { "message": "There may be a problem with the service or with your network connection." },
+  "loginErrorDetails": { "message": "Your shot was not saved. There was an error authenticating with the server." },
+  "loginConnectionErrorDetails": { "message": "There may be a problem with the service or your network connection." },
+  "unshootablePageError": { "message": "Page cannot be screenshotted." },
+  "unshootablePageErrorDetails": { "message": "This is not a normal web page, and Page Shot cannot capture screenshots from it." },
+  "recursivePageShotShotError": { "message": "You can't take a shot of a Page Shot page!" },
```

Comment:
I suggest to use proper apostrophes instead of straight quotes.
```
You can’t take a shot of a Page Shot page!
```

For reference, such a string would fail tests on Firefox and be rejected.